dspace-services/pom.xml

```diff
@@ -140,13 +140,20 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+	    <version>${log4j.version}</version>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+	    <artifactId>log4j-core</artifactId>
+	    <version>${log4j.version}</version>
         </dependency>
+	<dependency>
+	    <groupId>org.apache.logging.log4j</groupId>
+	    <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
dspace-services uses SLF4J exclusively.  Now that we have upgraded to Log4J v2 we can convert to that and remove some layers of logging frameworks.

I've been using SLF4J wherever I needed parameterized messages, since it's more efficient than multiple String concatenations.  Log4J2 has an even better version that accepts lambdas as message parameters.  I plan to use this quite a bit.